apps/camera_pipe/camera_pipe.cpp

```diff
@@ -15,13 +17,18 @@ Expr avg(Expr a, Expr b) {
 }
 
 Func hot_pixel_suppression(Func input) {
+    // hot pixel maximum
     Expr a = max(max(input(x-2, y), input(x+2, y)),
                  max(input(x, y-2), input(x, y+2)));
-    Expr b = min(min(input(x-2, y), input(x+2, y)),
-                 min(input(x, y-2), input(x, y+2)));
+    // cold pixel minimum
+    // Expr b = min(min(input(x-2, y), input(x+2, y)),
```

Comment:
I think the cold pixel suppression was unintentional (probably added since there were min operations performed in the original FCam...between "here" and the "max neighbors").  Suppressing cold pixels significantly degrades the contrast/overally image quality giving it a muddy/flat appearance.  I would recommend removing the code that computes the min of the neighbors from the intrinsic version as well.